reinforcement_learning/Lesson-5-A2C-draft.ipynb

```diff
@@ -0,0 +1,1017 @@
+{
```

Comment:
<p>rl-baseline3-zoo seems like a great tool for jumping in and showing the students how the challenge and rl model. That said, we still should add a section of an A2C model defined in code (can use the stable_baseline3 library I think)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BreakoutMentors/Data-Science-and-Machine-Learning/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>